Integrations/IntezerV2/IntezerV2.py

```diff
@@ -0,0 +1,252 @@
+        analysis_result = handle_analysis_result(response)
+
+        if analysis_result and analysis_type == 'Endpoint':
+            enrich_dbot_and_display_endpoint_analysis_results(analysis_result, demisto.getArg('indicator_name'))
+        elif analysis_result and analysis_type == 'File':
+            enrich_dbot_and_display_file_analysis_results(analysis_result)
+
+
+def make_analysis_status_request(analysis_id, analysis_type):
+    result_url = get_result_url(analysis_type, analysis_id)
+    session = get_session()
+    return session.get(result_url)
+
+
+def get_result_url(analysis_type, analysis_id):
+    result_url = '{}/{}/{}{}'.format(SERVER_URL, API_VERSION,
+                                         'endpoint-analyses/' if analysis_type == 'Endpoint' else 'analyses/',
+                                         analysis_id)
+
+    return result_url
+
+
+def handle_analysis_result(response):
+    json_response = handle_response(response, ACCEPTABLE_HTTP_CODES)
+
+    if response.status_code != 200:
+        result_url = json_response['result_url']
+        analysis_id = result_url.rsplit('/', 1)[-1]
+
+        context_json = {'Intezer.Analysis(val.ID === obj.ID)': {'ID': analysis_id,
+                                                                'Status': 'InProgress'}}
+
+        return_outputs('Analysis is still in progress', context_json)
+        return
+
+    return json_response['result']
+
+
+def enrich_dbot_and_display_file_analysis_results(result):
+    verdict = result['verdict']
+    sha256 = result['sha256']
+    analysis_id = result['analysis_id']
+
+    dbot = {
+        'Vendor': 'Intezer',
+        'Type': 'hash',
+        'Indicator': sha256,
+        'Score': dbot_score_by_verdict.get(verdict, 0)
+    }
+
+    file = {'SHA256': sha256, 'Metadata': result, 'ExistsInIntezer': True}
+
+    if verdict == 'malicious':
+        file['Malicious'] = {'Vendor': 'Intezer'}
+
+    presentable_result = '## Intezer File analysis result\n'
+    presentable_result += ' SHA256: {}\n'.format(sha256)
+    presentable_result += ' Verdict: **{}** ({})\n'.format(verdict, result['sub_verdict'])
+    if 'family_name' in result:
+        presentable_result += 'Family: **{}**\n'.format(result['family_name'])
+    presentable_result += '[Analysis Link]({})\n'.format(result['analysis_url'])
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'EntryContext': {outputPaths['dbotscore']: [dbot],
+                         outputPaths['file']: [file],
+                         'Intezer.Analysis(val.ID === obj.ID)': {'ID': analysis_id,
+                                                                 'Status': 'Done'}},
+        'HumanReadable': presentable_result,
+        'ContentsFormat': formats['json'],
+        'Contents': result
+    })
+
+
+def enrich_dbot_and_display_endpoint_analysis_results(result, indicator_name=None):
+    verdict = result['verdict']
```

Comment:
These accesses directly by keys in the dictionary are risky, are we certain these will be there?
I prefer we use `.get()`